doc/lsst.ap.pipe/getting-started.rst

```diff
@@ -13,83 +15,44 @@ Installation
-   may change when `DM-5467 <https://jira.lsstcorp.org/browse/DM-5467>`_ is completed.
+.. TODO: fill in details once we know what happens with image-like calibs
 
 
 .. _section-ap-pipe-required-data-products:
 
 Required data products
 ======================
 
-For the AP Pipeline to successfully process data, the following is required:
-
-- **Raw science images** and `reference catalogs 
-  <https://community.lsst.org/t/creating-and-using-new-style-reference-catalogs/1523>`_
-  ingested into a main Butler repository
+For the AP Pipeline to successfully process data, the following must be present in a Butler repository:
 
-  - The reference catalogs must be in a directory called :file:`ref_cats` with subdirectories
-    for each catalog containing the appropriate catalog shards.
-    We recommend using Pan-STARRS for photometry and Gaia for astrometry.
-    An example :ref:`config file <command-line-task-config-howto-configfile>` for using these two catalogs can be found in the `ap_verify_hits2015`_ repository.
+- **Raw science images** to be processed.
 
-- **Calibration products** (biases, flats, and defects, if applicable)
-  ingested into a Butler repository you must specify with the ``--calib`` flag on
-  the command line at runtime
+- **Reference catalogs** covering at least the area of the raw images.
+  We recommend using Pan-STARRS for photometry and Gaia for astrometry.
 
-  - To check if this requirement has been satisfied, you can inspect the
-    :file:`calibRegistry.sqlite3` created in this repository and ensure the information
-    in the flat, bias, and defect tables is accurate
+- **Calibration products** (biases, flats, and possibly others, depending on the instrument)
 
-- **Template images** (of type ``deepCoadd`` by default) for difference imaging
-  must be either in the main Butler repository or in another location you may
-  specify with the ``--template`` flag on the command line at runtime
+- **Template images** for difference imaging.
+  These are of type ``deepCoadd`` by default, but the AP pipeline can be configured to use other types.
 
 .. TODO: update default for DM-14601
 
 .. _ap_verify_hits2015: https://github.com/lsst/ap_verify_hits2015/
 
-A sample dataset from the `DECam HiTS survey <http://iopscience.iop.org/article/10.3847/0004-637X/832/2/155/meta>`_ 
-that works with ``ap_pipe`` in the :doc:`/modules/lsst.ap.verify/datasets` format
-is available as `ap_verify_hits2015`_. However, this dataset must be
-ingested as described in :ref:`section-ap-pipe-ingesting-data-files`, and the reference
-catalog and defect files must be decompressed and extracted.
+A sample dataset from the `DECam HiTS survey <http://iopscience.iop.org/article/10.3847/0004-637X/832/2/155/meta>`_ that works with ``ap_pipe`` in the :doc:`/modules/lsst.ap.verify/datasets` format is available as `ap_verify_hits2015`_.
+However, raw images from this dataset must be ingested as described in :ref:`section-ap-pipe-ingesting-data-files`.
```

Comment:
There's not much of a description linked here... does the ap_verify ingester work with gen3, and could it be pointed to here? If not, just cut the "as described in..."

Reply:
Not yet, it doesn't (still trying to figure out the Gen 3 system). I'll remove it.